config/routes.rb

```diff
@@ -26,6 +26,7 @@
   get "/sign-in", to: "sessions#create", as: :new_govuk_session
   get "/sign-in/callback", to: "sessions#callback", as: :new_govuk_session_callback
   get "/sign-out", to: "sessions#delete", as: :end_govuk_session
+  get "/save-a-page", to: "save_pages#create", as: :save_page
```

Comment:
Let's hold off on merging this until the discussion about paths in Slack has an answer.  If `/account/saved-pages` is the index, then we could have `/account/saved-pages/add` and `/account/saved-pages/remove` for manipulation.

Reply:
So it sounds like `/account/saved-pages` is going to be the index, so how about making the add / delete routes subpaths of that?

Reply:
done!